.github/workflows/deployment-report.js

```diff
@@ -0,0 +1,12 @@
+module.exports = async ({ github, context }) => {
```

Comment:
given this is used only in one place, is it better to inline it in the yml file instead?

Reply:
The only real reason to keep this separate was so that linters can properly apply to the js inside the action.

Reply:
if it's used a lot across repos, it sounds worth creating a separate actions repo? Something like https://github.com/iterative/get-pr-number. Then usage would be:

```yml
- id: pr-number
  uses: iterative/get-pr-number@v1
...
- run: echo ${{ steps.pr-number.outputs.result }}
```